src/payment/strategies/square/square-payment-strategy.ts

```diff
@@ -168,4 +301,12 @@ export interface SquarePaymentInitializeOptions {
      * The set of CSS styles to apply to all form fields.
      */
     inputStyles?: Array<{ [key: string]: string }>;
+
+    // Initialize Masterpass placeholder ID
```

Comment:
Any reason why this comment doesn't follow the format of the rest of the comments in this file?

Reply:
👍